index.html

```diff
@@ -1,42 +1,58 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
- <link rel="stylesheet" href="style.css" />
-  <title>Portfolio</title>
-</head>
-<body>
-
-	<header class="navbar">
-		 <div class="mylogo">
-		<a href="#" >Elhama Tokhi</a>
-	    </div>
-		<nav class="icon">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <header class="navbar">
+      <div class="mylogo">
+        <a href="#">Elhama Tokhi</a>
+      </div>
+      <nav class="icon">
         <img src="Images/Union.png" alt="Menu-icon" />
-    	 </nav>
+      </nav>
     </header>
 
+    <main class="container">
+      <h1>
+        I'm Elhama Tokhi 
+        Glad to see 
+        you!
+      </h1>
+      <p>
```

Comment:
- [ ] Kindly use a `section` tag here instead of a `main` tag since it would be more befitting for the project. Section tag is used to group together related elements and is used to distinguish between the different parts of the page. In the near future you would create the works section and the about section so it would be better to make this a headline section to keep the general idea of grouping related content in a `section`